### PR TITLE
Created routes and resolved database issue

### PR DESCRIPTION
**Pull request by ejchristie**
These changes should prevent the creation of multiple database instances. Also note that I added a third role called root (it was mentioned in the API blueprint).

### CODE REVIEW

File: routes/grading-url.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+var express = require('express');
+var router = express.Router();
+
+/* retrieve assignment metadata for a course */
+router.route('/getUrlToGrade/?assignment=:assignment_id')   // not sure if this is a valid way to do this
```
Comment (stuartf) [inline routes/grading-url.js]: It's either going to be:

    router.get('/getUrlToGrade', function(req, res) {
        var assignment_id = req.query.assignment;

or

    router.get('/getUrlToGrade/:assignment', function(req, res) {
        var assignment_id = req.param.assignment;

File: routes/assignments.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+router.route('/assignments')
+  .get();
+
+/* Make Submission Assignment */
+router.route('/assignments')
+  .post();
+
+/* Make Evaluation Assignment */
+router.route('/assignmentsTHISMAKESITNOTDUPLICATE')
+  .post();
```
Comment (ejchristie) [inline routes/assignments.js]: I just copied the paths from the API blueprint, but I can change it in both places.

File: db/sqlite.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+db.run("INSERT INTO role_map (role_id, role) VALUES (2, 'student')");
+
+module.exports = db;
+
+console.log('Current Module: ' + module);
+console.log('Main Module: ' + require.main);
+console.log('Module Children: ' + module.children);
+console.log('Module Exports: ' + module.exports);
+console.log('Module Parent: ' + module.parent);
+console.log('Is module loaded? ' + module.loaded);
```
Comment (ejchristie) [inline db/sqlite.js]: I think I deleted that file.

Comment (stuartf) [inline db/sqlite.js]: Looks like it, so that's not a problem anymore.

**Pull request merged**